go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/tetratelabs/wazero
 
 // Floor Go version of wazero (current - 2)
-go 1.17
+go 1.18
```

Comment:
this will hold us back until feb/when go 1.20 is released. are you anticipating keeping this PR open until then, or do you want to work around it for now?

Reply:
Ah OK, I may have misunderstood your directions on Slack.

Without this change Go refuses to build the package since generics weren't supported in 1.17. I guess I'll change the type to use concrete implementation and we'll keep copies for now then 👍